data_driven_onboarding/schema.py

```diff
@@ -73,9 +73,16 @@ class DDOSuggestionsResult(graphene.ObjectType):
     )
 
     hpd_open_violation_count = graphene.Int(
+        required=True,
         description=(
-            "Number of open HPD violations for the BBL. If there are no listed violations, "
-            "this will be null."
+            "Number of open HPD violations for the BBL."
+        )
+    )
+
+    hpd_open_class_c_violation_count = graphene.Int(
+        description=(
+            "The number of hpd violations associated with entered bbl that are "
+            "class C violations (since 2010)."
```

Comment:
Since this isn't marked as a `required` field, it would be good to explicitly document what is meant if this field is NULL.  On the other hand, if it's never NULL, we should mark it with `required=True`!